content/articles/2018-06-bench-1.0.1.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: bench 1.0.1
+date: '2018-06-28'
+slug: bench-1.0.1
+author: Jim Hester
+categories: [package]
+description: >
+  bench 1.0.1 is now available on CRAN. bench allows you to benchmark
+  code, tracking execution time, memory allocations and garbage
+  collections.
+photo:
+  url: https://unsplash.com/photos/nIkuMWT4Imc
+  author: Rula Sibai
+---
+
+[bench] is now available on CRAN!
+
+The goal of [bench] is to benchmark code, tracking execution time,
+memory allocations and garbage collections.
+
+Install the latest version with:
+``` r
+install.packages("bench")
+```
+
+### Comparison vs existing methods
+
+Compared to other methods such as [system.time], [rbenchmark], [tictoc] or
+[microbenchmark] we feel it has a number of benefits.
+
+- Always uses the highest precision APIs available for each operating system (often nanosecond-level).
+- Tracks memory allocations for each expression benchmarked.
+- Tracks the number and type of R garbage collections per expression iteration.
+- Verifies equality of expression results by default, to avoid accidentally
+  benchmarking inequivalent code.
+- `bench::press()`, a function which allows you to easily perform and combine
+  benchmarks across a large grid of values.
+- Uses adaptive stopping by default, running each expression for a set amount
+  of time rather than for a specific number of iterations.
+- Expressions are run in batches and summary statistics are calculated after
```

Comment:
Runs expressions in batches and calculates summary statistics… (again, list agreement)